src/Components/Tabs/Tabs.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './Tabs.css';
+import PropTypes from 'prop-types';
+
+export const Tabs = ({ tabs, selectedTabId, selectTabHandler }) => (
+  <div className="Tabs">
+    <ul className="Tabs__list">
+      {tabs.map(tab => (
+        <li
+          key={tab.id}
+          className={
+            selectedTabId === tab.id
+              ? 'Tab__item_active'
+              : null
+            }
```

Comment:
```suggestion
          className={
            selectedTabId === tab.id
              ? 'Tab__item_active'
              : null
            }
```
Можно использовать библиотеку Classnames.

Reply:
Не можно, а нужно 😄